samcli/local/init/templates/cookiecutter-aws-sam-hello-golang/README.md

```diff
@@ -1,49 +1,19 @@
-# Cookiecutter SAM for golang Lambda functions
+# Cookiecutter Golang Hello-world for SAM based Serverless App
 
-This is a [Cookiecutter](https://github.com/audreyr/cookiecutter) template to create a Serverless App based on Serverless Application Model (SAM).
-
-It is important to note that you should not try to `git clone` this project but use `cookiecutter` CLI instead as ``\\{\\{cookiecutter.project_slug\\}\\}`` will be rendered based on your input and therefore all variables and files will be rendered properly.
+A cookiecutter template to create a NodeJS Hello world boilerplate using [Serverless Application Model (SAM)](https://github.com/awslabs/serverless-application-model).
 
 ## Requirements
 
-Install `cookiecutter` command line:
-
-**Pip users**:
-
-* `pip install cookiecutter`
-
-**Homebrew users**:
-
-* `brew install cookiecutter`
-
-**Windows or Pipenv users**:
-
-* `pipenv install cookiecutter`
-
-**NOTE**: [`Pipenv`](https://github.com/pypa/pipenv) is the new and recommended Python packaging tool that works across multiple platforms and makes Windows a first-class citizen.
+* [AWS SAM CLI](https://github.com/awslabs/aws-sam-cli)
```

Comment:
Should this point to our landing page instead: https://aws.amazon.com/serverless/sam/